sample/github/README.md

```diff
@@ -6,156 +6,156 @@ deployed to receive the webhook's event deliveries and forward them into a
-stringData:
-  githubCredentials: >
-    {
-      "accessToken": "asdfasfdsaf",
-      "secretToken": "password"
-    }
-```
-
-## Running
-
-In response to a pull request event, the _legit_ Service will add `(looks pretty legit)` to the PR title.
-
-Deploy the _legit_ service via:
-
-```shell
-ko apply sample/github/legit-service.yaml
-```
+In response to a pull request event, the sample app _legit_ Service will add 
+`(looks pretty legit)` to the PR title.
 
-Once deployed, you can inspect the created resources with `kubectl` commands:
-
-```shell
-# This will show the Service that we created:
-kubectl get service.serving.knative.dev -o yaml
-
-# This will show the Route that the Service created:
-kubectl get route -o yaml
-
-# This will show the Configuration that the Service created:
-kubectl get configurations -o yaml
+## Prerequisites
 
-# This will show the Revision that was created by the Configuration:
-kubectl get revisions -o yaml
-```
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com/) installed and running on your local machine,
+  and a Docker Hub account configured (you'll use it for a container registry).
+- The core Knative eventing tools installed. You can install them with:
+  ```shell
+  kubectl apply -f https://storage.googleapis.com/knative-releases/eventing/latest/release.yaml
+  ```
+- A custom domain is setup to allow for GitHub to be able to call into the cluster:
+  [configure a custom domain](https://github.com/knative/docs/blob/master/serving/using-a-custom-domain.md) and 
+  [assign a static IP address](https://github.com/knative/docs/blob/master/serving/gke-assigning-static-ip-address.md).
 
-The `Flow` will accept the Webhook calls from GitHub and pass them to the _legit_ service.
+## Configuring Knative
 
-You can inspect those resources with the following `kubectl` commands:
+To use this sample, you'll need to install the `stub` ClusterBus and the 
+`github` EventSource.
 
 ```shell
-# This will show the available EventSources that you can generate feeds from:
-kubectl get eventsources -o yaml
-
-# This will show the available EventTypes that you can generate feeds from:
-kubectl get eventtypes -o yaml
+kubectl apply -f https://storage.googleapis.com/knative-releases/eventing/latest/release-clusterbus-stub.yaml
+kubectl apply -f https://storage.googleapis.com/knative-releases/eventing/latest/release-source-github.yaml
 ```
 
-Configure the `Flow` to point to a GitHub repository you are able to control and update
-`spec.trigger.resource` in `samples/github/flow.yaml`.  
-
-And then apply the resources:
+## Granting permissions
 
-```shell
-ko apply -f sample/github/auth.yaml
-ko apply -f sample/github/flow.yaml
-```   
+Because the `github` EventSource needs to create a
+`Service.serving.knative.dev`, you'll need to provision a special
```

Comment:
I'd recommend using the non-fully scoped name here: `EventSource needs to create a Knative Service`